.env

```diff
@@ -0,0 +1 @@
+VITE_DEVCYCLE_CLIENT_SDK_KEY=dvc_client_ead08ba9_a12b_424d_a4c3_a329cf940f74_aa8e54b
```

Comment:
looks like .env is missing in the gitignore?

Reply:
yikes good catch — now lets see if this token is flagged as exposed in the dashboard 👀